src/styles/blocks/card.scss

```diff
@@ -0,0 +1,125 @@
+@import "../partials/variables.scss";
+
+.card {
+  max-width: 198px;
+  min-height: 406px;
+  transition: all 300ms;
+  &:hover {
+    transform: scale(1.2);
+  }
+  &__content {
+    width: $card__content-width;
+    border: 1px solid $card-border-color;
+    border-radius: 5px;
+    box-sizing: border-box;
+    padding-block: auto;
+    &-image {
+      width: $card_image-size-width;
+      height: $card_image-size-height;
+      margin-top: 32px;
+      margin-left: 18px;
+    }
+  }
+
+  &__title {
+    font-size: 12px;
+    width: $card-element-width;
+    height: 36px;
+    margin-top: 36px;
+    margin-left: 15px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+    display: flex;
+    padding-right: 20px;
+    transition: all 300ms;
```

Comment:
the transition should be more specific, please target only CSS properties you will be animating